packages/dropdowns/src/types/index.ts

```diff
@@ -185,6 +185,8 @@ export interface IOptGroupProps extends Omit<LiHTMLAttributes<HTMLLIElement>, 'c
   /** Accepts an icon to display */
   icon?: ReactElement;
   /** Sets the text label of the option group */
+  legend?: string;
+  /** @deprecated use `IOptGroupProps['legend']` instead */
```

Comment:
I was looking around in the gatsby prop sheet code and couldn't quickly see the best way to ensure `label` is marked deprecated or ignored at build time. I also saw `@ignore`, but that has a different meaning I guess.

Reply:
Per sequence doc, let's pull the bandaid off and remove `label` rather than deprecating.